app/models/country.rb

```diff
@@ -1,2 +1,7 @@
 class Country < ApplicationRecord
+  has_many :static_stat
```

Comment:
I noticed we have `has_many static_stat` in Country and `has_many static_stats` in Habitat. I just wondered about the pluralisation both with and without. Should they both be the same as each other?